src/main/java/tools/jackson/databind/JsonNode.java

```diff
@@ -991,6 +1014,15 @@ public Iterator<Map.Entry<String, JsonNode>> fields() {
         return ClassUtil.emptyIterator();
     }
 
+    /**
+     * @return <code>Spliterator</code> that can be used to traverse all key/value pairs
+     *   for object nodes; empty spliterator (no contents) for other types
+     * @since 3.0
+     */
+    public Spliterator<Map.Entry<String, JsonNode>> fieldsSpliterator() {
```

Comment:
3.0 uses term "property" everywhere, all references to "field" are changed.

But combination of property name and value, "entry" is used in `JsonNode`, see `entryStream()`.

So should be `entrySpliterator()`?

Reply:
Actually, let me think about this one: I may have chosen wrong term: instead "entry" should really use "property"/"properties, as per:

https://github.com/FasterXML/jackson-future-ideas/wiki/JSTEP-6

will change `entryStream` to `propertyStream()`; and then we should have `propertySpliterator`.
Or, if we are decide to go with plurals, `propertiesSpliterator()`.

Reply:
Singular it is: will change to `propertySpliterator()`. Similar for value case.